src/content/docs/usage/Node Graph/Tutorials/your-first-graph.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Your First Graph
-description: Learn how to create your first Node Graph in PixiEditor.
+title: Create a Node graph
```

Comment:
Changing the title changes the URL of the doc. You must include `slug: usage/node-graph/tutorials/your-first-graph` so the previous url doesn't lead to 404 and sidebar entry works

Reply:
Additionally capitalization of "graph" is inconsistent with the rest of the doc. I'd personally captialize "G"